app/reducers/captures.js

```diff
@@ -15,6 +15,28 @@ export function captures (state = {}, action) {
           }
         }
       };
+    case MARK_PENDING:
+      const pendingState = {
+        ...state,
+        [action.username]: {
+          ...state[action.username],
+          dexesBySlug: {
+            [action.slug]: {
+              ...state[action.username].dexesBySlug[action.slug],
+              captures: state[action.username].dexesBySlug[action.slug].captures.slice()
+            }
+          }
+        }
+      };
+      const pendingIndex = pendingState[action.username].dexesBySlug[action.slug].captures.findIndex((c) => c.pokemon.id === action.pokemon[0]);
+      for (let i = pendingIndex, count = 0; count < action.pokemon.length; i++) {
+        if (action.pokemon.indexOf(pendingState[action.username].dexesBySlug[action.slug].captures[i].pokemon.id) !== -1) {
+          pendingState[action.username].dexesBySlug[action.slug].captures[i].captured = action.captured;
```

Comment:
instead of passing in `captured: false` in the action, since it's always going to be false, it should probably hard-code `false` here, similarly to how `pending` is being hard-coded to `true`